brand/utils/harvest_data.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime, timedelta
+from urllib.parse import quote
+
+from django.conf import settings
+
+import requests
+
+
+def fetch_harvest_data(brand_tag, brand_url="", brand_country="", brand_name=""):
```

Comment:
I think it would be good to specify a return type here. IMO that would either be JSON or returning an exception. 

```
from typing import Union, Dict
def fetch_harvest_data(brand_tag, brand_url="", brand_country="", brand_name="") -> Union[Dict, Exception]:

    try:
        requests.get()
    except Exception as e:
        return e